tests/python/unittest/test_arith_rewrite_simplify.py

```diff
@@ -0,0 +1,249 @@
+import tvm
```

Comment:
I think that this kind of tests is not enough for such a complex algorithm. It is hard to be sure that existing test cases trigger every rule from the long list above. Probably we have to generate test cases with answers rather then hardcode them.